api/python/provisioner/api_spec.yaml

```diff
@@ -47,7 +47,7 @@ pillar_set:
   type: PillarSet
   input_type: PillarInputBase
 set_ntp:
-  type: Set
+  type: SetNTP
```

Comment:
Should this be SetControllerNTP?

Reply:
This API is not specific to the controller but for both nodes and the controller together.
So, whenever NTP is config command is run it will first set the NTP on nodes followed by controller.